go/private/rules/BUILD.bazel

```diff
@@ -0,0 +1,147 @@
+load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
```

Comment:
Adding a build file here and in other subdirectories of `//go/private` will break old versions of Gazelle.

It's probably the right thing to do, but it would be better to fix Gazelle, then not release this change for a while so people have some time to upgrade. People are much more likely to upgrade rules_go than Gazelle.

Reply:
This turned out to be a [relatively simple fix](https://github.com/bazelbuild/bazel-gazelle/pull/879) in gazelle. Hurray! Note that it won't pass until #2621 is merged and released.

Reply:
This is #2621; did you mean a different PR?

Reply:
Sorry, I meant https://github.com/bazelbuild/bazel-gazelle/pull/760

Reply:
Turns out we were able to resolve it none the less.